### PR TITLE
[Keyboard] Add pocketpad

### PR DESCRIPTION
**Pull request by butterkeebs**
## Description

Added the ButterKeebs PocketPad folder and relevant files

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/butterkeebs/pocketpad/info.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": false,
+        "sleep_led": false
+    },
+    "keyboard_folder": "butterkeebs/pocketpad",
+    "keymaps": {},
```
Comment (waffle87) [inline keyboards/butterkeebs/pocketpad/info.json:26]:
```suggestion
```

File: keyboards/butterkeebs/pocketpad/info.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        "rows": ["F7", "F6", "F5", "F4", "B1"]
+    },
+	"matrix_size": {
+        "cols": 4,
+        "rows": 5
+    },
+    "mouse_key": {
+        "enabled": true
+    },
+
```
Comment (waffle87) [inline keyboards/butterkeebs/pocketpad/info.json:38]:
```suggestion
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Comment by butterkeebs** [comment]
Folder ButterKeebs/PocketPad has been renamed to butterkeebs/pocketpad to comply with naming scheme

**Review by waffle87 — CHANGES_REQUESTED** [review]
The directory name(s) should also be all lowercase, as per the PR checklist.
Lint check is also failing.

**Comment by butterkeebs** [comment]
> The directory name(s) should also be all lowercase, as per the PR checklist. Lint check is also failing.

The directories are now lowercase? What is lint check?

**Comment by waffle87** [comment]
You've split up the directories...
There's [`keyboards/ButterKeebs/PocketPad`](https://github.com/butterkeebs/qmk_firmware/tree/master/keyboards/ButterKeebs/PocketPad) and [`keyboards/butterkeebs/pocketpad`](https://github.com/butterkeebs/qmk_firmware/tree/master/keyboards/butterkeebs/pocketpad).

**Review by waffle87 — APPROVED** [review]

**Comment by zvecr** [comment]
`readme.md` (following <https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md>) is currently considered mandatory.

**Review by drashna — APPROVED** [review]

**Review by drashna — CHANGES_REQUESTED** [review]
readme is needed.

File: keyboards/butterkeebs/pocketpad/readme.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<h1> PocketPad </h1>
```
Comment (zvecr) [inline keyboards/butterkeebs/pocketpad/readme.md:1]: This does not follow the template that was linked

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by waffle87 — APPROVED** [review]

**Pull request merged**